icevision/models/mmdet/common/mask/show_batch.py

```diff
@@ -0,0 +1,18 @@
+__all__ = ["show_batch"]
+
+from icevision.utils import *
+from icevision.visualize import *
+
+
+def show_batch(batch_and_records, ncols: int = 1, figsize=None, **show_samples_kwargs):
+    """Show a single batch from a dataloader.
+    # Arguments
+        show_samples_kwargs: Check the parameters from `show_samples`
+    """
+    batch, records = batch_and_records
+
```

Comment:
Do this function also needs:
```python
    if isinstance(batch["img"], list):
        batch = {
            "img": batch["img"][0],
            "img_metas": batch["img_metas"][0],
        }
```

If so, is this exactly the same as we have for box? We can maintain a single function under `common` and just export it from here